### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV Medications - Refill Alerts & Trackers

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#102732]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Lots of really great stuff here! 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** Per [design system guidance for the process list component](https://design.va.gov/components/process-list#additional-accessibility-considerations-for-va), make sure each item in the process list has content that follows the heading. Screen reader users often navigate the page skipping heading to heading, and then if they hear a heading that sounds like what they're looking for they'll have their screen reader announce the page content starting from that point. When there's no content that comes after a heading, users might conclude that something is broken or still loading.
   
   I realize some of the steps don't have obvious content to provide when they're not the active step. The content here can be short and perfunctory when appropriate, but it's important to provide something besides the heading.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
